File: Cliente.java
public class Cliente implements identificavel{
    private String nomeCLiente;
    private String cpf;
    private String email;
    private String profissao;
    private String telefone;
    private String endereco;

    public String getNomeCLiente() {
        return nomeCLiente;
    }

    public void setNomeCLiente(String nome) {
        this.nomeCLiente = nome;
    }

    public String getCpf() {
        return cpf;
    }

    public void setCpf(String cpf) {
        this.cpf = cpf;
    }

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }

    public String getProfissao() {
        return profissao;
    }

    public void setProfissao(String profissao) {
        this.profissao = profissao;
    }

    public String getTelefone() {
        return telefone;
    }

    public void setTelefone(String telefone) {
        this.telefone = telefone;
    }

    public String getEndereco() {
        return endereco;
    }

    public void setEndereco(String endereco) {
        this.endereco = endereco;
    }


        // 3) Criar interface

    @Override
    public String identifcar() {
        return "Nome: " + this.getNomeCLiente() + "\nCPF: "+ this.getCpf();
    }
}

File: Comex/Comex.Modelos/modelo/Cliente.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Comex.modelo;

public class Cliente : Identificavel
{
    public string Nome {  get; set; }
    public string Cpf { get; set; }
    public string Email { get; set; }
    public string Profissao { get; set; }
    public string Telefone { get; set; }
    public Endereco endereco { get; set; }


    public void Identificar()
    {
        Console.WriteLine("Nome do Cliente: " + Nome);
        Console.WriteLine("CPF do Cliente: " + Cpf);
    }
}
